router/session.py

```diff
@@ -0,0 +1,103 @@
+from fastapi import APIRouter, HTTPException
+import requests
+from settings import settings
+from datetime import datetime
+
+router = APIRouter(prefix="/session", tags=["Session"])
+session_db_url = settings.db_url + "session/"
+session_start_buffer_time = 900000
+
+def get_current_datetime():
+    """
+    Returns current datetime
+    """
+    return datetime.today()
+
+def get_current_date():
+    """
+    Returns current date
+    """
+    return get_current_datetime().date()
+
+def get_current_timestamp():
+    """
+    Returns current timestamp
+    """
+    return get_current_datetime().timestamp()
+
+def get_date(datetime: datetime):
+    """
+    Returns date for the given datetime
+    """
+    return datetime.date()
+
+def get_timestamp(datetime: datetime):
+    """
+    Returns timestamp for the given datetime
+    """
+    return datetime.timestamp()
+
+def is_start_time_valid(start_time: str, repeat_schedule: str):
+    """
+    Checks if session start time is valid
```

Comment:
can you also mention what do you mean by `valid` here? What does session start time validity means here

Reply:
Renamed to `has_session_started`